src/Stack/Package.hs

```diff
@@ -96,9 +96,14 @@ instance HasPantryConfig Ctx where
     pantryConfigL = configL.pantryConfigL
 instance HasProcessContext Ctx where
     processContextL = configL.processContextL
-instance HasBuildConfig Ctx
+instance HasBuildConfig Ctx where
+    buildConfigL = lens ctxBuildConfig (\x y -> x { ctxBuildConfig = y })
+    {-
+instance HasSourceMap Ctx where
+    sourceMapL = envConfigL.sourceMapL
 instance HasEnvConfig Ctx where
     envConfigL = lens ctxEnvConfig (\x y -> x { ctxEnvConfig = y })
+    -}
```

Comment:
Did you want to remove this commented out code?

Reply:
Good catch!